### PR TITLE
[CIRCLE-13826]: Periodic update check for user

### PR DESCRIPTION
**Pull request by zzak**

### CODE REVIEW

File: update/update.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		}
+
+		return false, errors.Wrap(err, "error finding latest release")
+	}
+
+	return found, nil
+}
+
+// NewerVersionAvailable will tell us if the current version is the same as the latest version found from the GitHub releases API.
+func (options *Options) NewerVersionAvailable() bool {
```
Comment (dnephin) [inline update/update.go]: The name `NewerVersionAvailable` suggests to me that a true value would indicate that there are  updates available, however that is does not match the implementation.

Maybe `IsUpToDate` or `IsLatestVersion` ?

Comment (zzak) [reply update/update.go]: Went with `IsLatestVersion`, good catch :+1:

**Comment by zzak** [comment]
Holding off on merging this until we add some tests to it.

**Comment by zzak** [comment]
Holding off on merging this until we add some tests to it.

**Pull request merged**